src/main/scala/org/scalajs/tools/tsimporter/parser/TSDefParser.scala

```diff
@@ -112,6 +113,13 @@ class TSDefParser extends StdTokenParsers with ImplicitConversions {
   lazy val typeAliasDecl: Parser[DeclTree] =
     "type" ~> typeName ~ tparams ~ ("=" ~> typeDesc) <~ opt(";") ^^ TypeAliasDecl
 
+  lazy val importDecl: Parser[DeclTree] =
+    "import" ~> (
+        "{" ~ identifier ~ opt(lexical.Identifier("as") ~ identifier) ~ "}" ~ lexical.Identifier("from") ~ stringLiteral
+      | "*" ~ lexical.Identifier("as") ~ identifier ~ lexical.Identifier("from") ~ stringLiteral
+      | stringLiteral
```

Comment:
You're missing the default import syntax:
```js
import foo from "bar.js"
```

Reply:
It seems I shouldn't have just relied on the TypeScript docs, they missed some obvious examples and so did I. ;)